data/templates.json

```diff
@@ -4,31 +4,27 @@
     "objectType": "MPManuscriptTemplate",
     "bundle": "MPBundle:www-zotero-org-styles-american-medical-association",
     "title": "Nature Genetics Journal Publication",
-    "createdAt": 1433095330.89114,
-    "updatedAt": 1433095363.40453
+    "sectionCategories": "default-section-categories.json"
```

Comment:
- I think we should allow the value to be an array directly as well.
- If the value is a string, we should treat it as a path, and resolve that path relative to `DATA_PATH`.
- We should have the value be clearly a path, something like "@default-section-categories.json" or "./default-section-categories.json".